test/services/PageBuilderErrorsSpec.scala

```diff
@@ -310,7 +310,8 @@ class PageBuilderErrorsSpec extends BaseSpec with ProcessJson {
         "1" -> InstructionStanza(0, Seq("2"), None, false),
         "2" -> QuestionStanza(1, Seq(2, 3), Seq("4", "5"), None, false),
         "4" -> PageStanza("/session-restart", Seq("5"), false),
-        "5" -> PageStanza("/session-timeout", Seq("end"), false),
+        "5" -> PageStanza("/session-timeout", Seq("6"), false),
+        "6" -> PageStanza(s"${SecuredProcess.SecuredProcessStartUrl}", Seq("end"), false),
```

Comment:
I think you're missing the fore slash here as the url for stanza 6 should be s"/${SecuredProcess.SecuredProcessStartUrl}".